### DISCUSSION TITLE
Format of Questions

### DISCUSSION THREAD

**Discussion by oklev** [UGA-BIGSA/gsss#1] [Team Posts]
Hey so I took a peek at the qualtrics "QSF" format for importing and exporting surveys. It is technically readable but quite ugly -- probably not what we want to be working in. We could probably make something that converts our preffered format to the qualtrics format
<img width="1345" alt="Screen Shot 2023-04-18 at 1 39 54 PM" src="https://user-images.githubusercontent.com/116015506/232859514-22199c15-685d-49f1-ac39-88fc042b3b6e.png">
^QSF format

But it would probably be better not to. There's a [simple](https://www.qualtrics.com/support/survey-platform/survey-module/survey-tools/import-and-export-surveys/#PreparingASimpleFormatTXTFile) and [slightly more complicated but still pretty straightforward](https://www.qualtrics.com/support/survey-platform/survey-module/survey-tools/import-and-export-surveys/#PreparingAnAdvancedFormatTXTFile) format we could use for preparing the survey if that works for y'all.

**Comment by glstott** [comment]
Yup! I agree. I think that format should be totally sufficient. We can link to the instructions in the README. It may make it a bit more difficult to maintain, and I would worry about proprietary software standards changing on us long term though. I think this is our best bet.
